src/Network/HttpClient.php

```diff
@@ -102,26 +83,14 @@ private function getClient()
      * Sends request to Google Analytics.
      *
      * @param string $url
-     * @param SingleParameter[] $singleParameters
-     * @param CompoundParameterCollection[] $compoundParameters
      * @param boolean $nonBlocking
      * @return AnalyticsResponse
      */
-    public function post($url, array $singleParameters, array $compoundParameters, $nonBlocking = false)
+    public function post($url, $nonBlocking = false)
```

Comment:
Not directly, but multiple test cases failed since it was mocked the same way it was done here (which had to be changed). Just thought I'd check with you if this was really intended. Looks like I'll have to change these tests.

Anyway, thanks for the fast response!